src/include/block.h

```diff
@@ -299,6 +299,7 @@ struct __wt_block {
     uint64_t compact_pages_rewritten_expected; /* The expected number of pages to rewrite */
     uint64_t compact_pages_skipped;            /* Pages skipped */
     uint32_t compact_session_id;               /* Session compacting */
+    wt_off_t compact_prev_size;                /* File size at the start of compact */
```

Comment:
And I suggest to change the comment:
```suggestion
    wt_off_t compact_prev_size;                /* File size at the start of a compaction pass */
```